### PR TITLE
Copied code

### PR DESCRIPTION
**Pull request by ooshoke**

### CODE REVIEW

**Review by ooshoke — COMMENTED** [review]
Add linters to this project. For a guide on how to do that, look here https://github.com/microverseinc/linters-config/blob/master/html-css

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles.css">
+    <title>Document</title>
+    <style>
```
Comment (ooshoke) [inline index.html:9]: I think it would be better to move style rules to the styles.css file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        
+        @media only screen and (max-width: 600px) {
+          body {
+            background-color: lightblue;
+          }
+        }
+        </style>
+</head>
+<body>
+    <header id="header">
```
Comment (ooshoke) [inline index.html:22]: use a class identifier as opposed to an id and also, choose a better id value.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+         <nav id="nav-bar"> 
+             <button type="button"> <a href="#tail" class="nav-link">top</a> </button>
+             <button type="button"> <a href="#footer" class="nav-link">bottom</a> </button>
+             <button type="button"> <a href="#bodies" class="nav-link">nav</a> </button>
+
+         </nav>
+          
+    </header>
+    <div id="footer"></div>
+    <div id="bodies"></div>
```
Comment (ooshoke) [inline index.html:33]: Make use of semantic tags

**Review by ooshoke — COMMENTED** [review]
Please review the changes required and configure linters

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @ooshoke,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Review Trials Project PR or Review of Past  Project".
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
- [ ] Kindly make sure in every review you do tell the student the reason why you are commenting on a particular line of code.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**